examples/pony/alerts_windowed/README.md

```diff
@@ -0,0 +1,94 @@
+# Alerts (stateless)
+
+## About The Application
+
+This is an example of a stateless application that takes a transaction
+and sends an alert if its value is above or below a threshold.
+
+### Input
+
+For simplicity, we use a generator source that creates a stream of transaction
+objects.
+
+### Processing
+
+We examine each transaction at a stateless computation. If the amount is higher
+than a certain positive threshold, we create a deposit alert. If it's lower than a certain negative threshold, we create a withdrawal alert.
+
+### Output
+
+Alerts will output messages that are string representations of triggered
+alerts. 
+
+## Running Alerts
+
+In order to run the application you will need Machida, Data Receiver, and the Cluster Shutdown tool. We provide instructions for building these tools yourself and we provide prebuilt binaries within a Docker container. Please visit our [setup](https://docs.wallaroolabs.com/book/getting-started/choosing-an-installation-option.html) instructions to choose one of these options if you have not already done so.
```

Comment:
You don't need machida for the pony example

Reply:
addressed